leases/schema.py

```diff
@@ -119,12 +124,10 @@ class Input:
         id = graphene.ID(required=True)
 
     @classmethod
-    @login_required
-    @superuser_required
+    @view_permission_required(BerthApplication)
```

Comment:
well, you do not have access to the application's info through this mutation, so no need to check for this